sw/device/lib/dif/autogen/dif_hmac_autogen_unittest.cc

```diff
@@ -82,16 +82,14 @@ TEST_F(IrqIsPendingTest, Success) {
 
   // Get the first IRQ state.
   irq_state = false;
-  EXPECT_READ32(HMAC_INTR_STATE_REG_OFFSET,
-                {{HMAC_INTR_STATE_HMAC_DONE_BIT, true}});
+  EXPECT_READ32(HMAC_INTR_STATE_REG_OFFSET, {{0, true}});
```

Comment:
No, this had to be touched. Specifically, the unit tests had to be more generic across all IPs since the RV Timer does not have an autogenerated macro that matches the pattern: `HMAC_INTR_STATE_<IRQ name>_BIT` (same for enable and test regs). This is because the RV timer has IRQ names that, in theory, could span multiple harts/timers, and each hart has a separate set of `ENABLE/STATE/TEST` regs.

Does that make sense?

Reply:
Right, but....all those other IPs that are 'dull' (i.e. have intr state/enable/test CSRs), such as HMAC need not change right? Its just the RV timer autogen'ed DIF that needs to be this way. 

Replacing `HMAC_INTR_STATE_HMAC_DONE_BIT` which is defined in `hmac_regs.h` with a hard coded value here does not seem right. Sorry I maybe missing the point here.